src/helpers/respond.ts

```diff
@@ -0,0 +1,11 @@
+import Express from 'express'
+
+const respond = (statusCode: number, data: any, res: Express.Response) => {
+  res.status(statusCode).json({
+    status: true,
+    message: 'success',
```

Comment:
message with `success` value is added to have an identical schema, similar to the one which's used in error responses (the message in error responses will be used to show the error message)

Reply:
Sure, it would be more meaningful also I think. and we can also use success with value `false` when responding with errors.

Reply:
Changes are committed.